apps/web/src/app/layout.tsx

```diff
@@ -11,10 +11,10 @@ import { Metadata } from "next";
 import socialpreview from "@/assets/social-preview.png";
 
 export const metadata: Metadata = {
+  metadataBase: new URL(process.env["NEXT_PUBLIC_METADATA_FRONTEND_URL"]),
```

Comment:
```suggestion
  metadataBase: new URL(process.env["NEXT_PUBLIC_FRONTEND_URL"]),
```

zgodnie z powyższym, tu też by trzeba było zmienić :)